db/migrations/20220104165950_goteo_data_sets.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Migration Task class.
+ */
+class GoteoDataSets
+{
+  public function preUp()
+  {
+      // add the pre-migration code here
+  }
+
+  public function postUp()
+  {
+      // add the post-migration code here
+  }
+
+  public function preDown()
+  {
+      // add the pre-migration code here
+  }
+
+  public function postDown()
+  {
+      // add the post-migration code here
+  }
+
+  /**
+   * Return the SQL statements for the Up migration
+   *
+   * @return string The SQL string to execute for the Up migration.
+   */
+  public function getUpSQL()
+  {
+     return "CREATE TABLE `data_set` (
+            `id` BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT,
+            `title` TEXT NULL,
+            `description` VARCHAR(10) NULL,
+            `lang` VARCHAR(6) NOT NULL,
+            `url` VARCHAR(10) NOT NULL,
+            `image` VARCHAR(255) NULL,
+            `created` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+            `modified` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
```

Comment:
The convention for created and modified fields, is to write them as "created_at" and "modified_at". Or "created_on" and "modified_on". 

More info:https://stackoverflow.com/questions/1379543/database-name-convention-datetime-column